test/powershell/Language/Scripting/ForeachParallel.Tests.ps1

```diff
@@ -2,19 +2,12 @@
 # Licensed under the MIT License.
 Describe "Parallel foreach syntax" -Tags "CI" {
 
-   Context 'Should be able to retrieve AST of parallel foreach, error in regular case' {
-        $errors = @()
+    Context 'Should be able to retrieve AST of parallel foreach' {
         $ast = [System.Management.Automation.Language.Parser]::ParseInput(
-            'foreach -parallel ($foo in $bar) {}', [ref] $null, [ref] $errors)
-        It '$errors.Count' { $errors.Count | Should -Be 1 }
+            'foreach -parallel ($foo in $bar) {}', [ref] $null, [ref] $null)
```

Comment:
Please put this in BeforeAll.
Below too.

Reply:
Fixed.